include/grpcpp/server_context.h

```diff
@@ -291,9 +332,7 @@ class ServerContextBase {
   /// client in order to make load balancing decisions. This will
   /// return nullptr if the feature hasn't been enabled using
   /// \a EnableCallMetricRecording.
-  experimental::CallMetricRecorder* ExperimentalGetCallMetricRecorder() {
-    return call_metric_recorder_;
-  }
+  CallMetricRecorder* GetCallMetricRecorder() { return call_metric_recorder_; }
```

Comment:
Please do not remove the `Experimental` prefix from this method name.  We want to keep this API experimental for now.

Reply:
Done.